src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
+
+        System.out.println("Ball 1 = " + ball1);
+        System.out.println("Ball 2 = " + ball2);
+        System.out.println("Ball 3 = " + ball3);
```

Comment:
use for-loop for this

Reply:
fix this
what if we'd have to create 100500 balls?